pp.c

```diff
@@ -2451,6 +2460,15 @@ PP(pp_seq)
     SV *right = PL_stack_sp[0];
     SV *left  = PL_stack_sp[-1];
 
+    if(PL_op->op_private & OPpEQ_UNDEF) {
```

Comment:
The numeric ops use UNLIKELY(), the string ops don't, is there a reason?

Reply:
Oops, no. Just laziness on my part. I added them to the numerical and forgot the string. I shall make consistent.